lib/SILGen/SILGenApply.cpp

```diff
@@ -174,7 +174,10 @@ class Callee {
   const Kind kind;
 
   // Move, don't copy.
+  // Work around MSVC error: attempting to reference a deleted function.
+#if !defined(_MSC_VER) || defined(__clang__)
```

Comment:
You might want to take a look at https://github.com/apple/swift/pull/5948 in which I try to work around a similar error. Unfortunately, this time the code that triggers this is compiler generated `__delDtor(unsigned int)`